test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+import seedu.addressbook.common.Utils;
+
+public class UtilsTest {
+    Utils utils = new Utils();
+    
+    @Test
+    public void testIsNull() {
```

Comment:
Note that our naming convention for test methods is as follows:
`[feature under test/method name]_[scenario]_[expected result]`
e.g. `isAnyNull_emptyList_falseReturned`